package_managers/apt_get/repos.bzl

```diff
@@ -49,3 +51,16 @@ def generate_additional_repos(name, repos):
 Args:
   repos: List of repos to add in sources.list format.
 """
+
+def add_gpg_key_from_url(name, image, gpg_url):
```

Comment:
The name does not convey this is going to produce a docker image.
I checked, apt get install does not download a package if its already installed. 
This means, if we run download_pkgs with this image, packages curl and gpg cannot be fetched.